content/en/docs/ops/configuration/traffic-management/network-topologies/index.md

```diff
@@ -1,6 +1,6 @@
 ---
-title: Configuring Gateway Network Topology [Alpha]
-description: How to configure gateway network topology (alpha).
+title: Configuring Gateway Network Topology (Alpha)
```

Comment:
@howardjohn @jacob-delgado I think this should probably say "Experimental"?

Reply:
I know this was moved to Alpha with https://github.com/istio/istio.io/pull/9556 and https://github.com/istio/enhancements/pull/37.

Reply:
👍